### PR TITLE
usestate-again: Some tweaks

### PR DESCRIPTION
**Pull request by sophiebits**
On top of https://github.com/reactjs/reactjs.org/pull/4284.

### CODE REVIEW

File: beta/src/pages/reference/usestate.md [hunk 1]
```diff
@@ -68,17 +58,13 @@ function Form() {
 
 ---
 
-### `setState(nextState)` {/*setstate*/}
+### `set` functions, like `setName(nextState)` {/*setstate*/}
```
Comment (gaearon) [inline beta/src/pages/reference/usestate.md:61]: is it weird to say setName(nextState)?

File: beta/src/pages/reference/usestate.md [hunk 2]
```diff
@@ -68,17 +58,13 @@ function Form() {
...
 
-`setState` lets you update the state to a different value and trigger a re-render:
+The `set` function returned by `useState` lets you update the state to a different value and trigger a re-render. You can pass the next state directly, or a function that calculates it from the previous state:
 
 ```js
-setState(nextState);
-```
-
-You can pass the next state directly, or a function that calculates it from the previous state:
+const [name, setName] = useState('Edward');
```
Comment (gaearon) [inline beta/src/pages/reference/usestate.md:66]: 👀

File: beta/src/pages/reference/usestate.md [hunk 3]
```diff
@@ -92,13 +78,13 @@ function handleClick() {
 
 #### Returns {/*setstate-returns*/}
 
-`setState` does not have a return value.
+`set` functions do not have a return value.
 
 #### Pitfalls {/*pitfalls*/}
 
 * Calling `setState()` only updates the state for the next render. If you read the `state` variable after calling `setState()`, you will still get the value that was on the screen before your call.
```
Comment (gaearon) [inline beta/src/pages/reference/usestate.md:85]: Should we not call it `setState()` here?

File: beta/src/pages/reference/usestate.md (hunk 2 above)
Comment (sophiebits) [reply beta/src/pages/reference/usestate.md:66]: You used Taylor for the initial state and the update so I had to make a name up!

File: beta/src/pages/reference/usestate.md (hunk 1 above)
Comment (sophiebits) [reply beta/src/pages/reference/usestate.md:61]: 🤷🏻‍♀️

File: beta/src/pages/reference/usestate.md (hunk 3 above)
Comment (sophiebits) [reply beta/src/pages/reference/usestate.md:85]: 🤷🏻‍♀️

**Pull request merged**